akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -26,6 +54,10 @@ object ErrorInfo {
    * Used for example when catching exceptions generated by the header value parser, which doesn't provide
    * summary/details information but structures its exception messages accordingly.
    */
+  private[akka] def apply(summary: String = "", detail: String = ""): ErrorInfo = new ErrorInfo(summary, detail, "")
```

Comment:
You introduced these new methods between the existing scaladoc documentation and the method they document below - can you reorder that?

Reply:
I made a mistake.  reorder methods. thanks.